### PR TITLE
[master] Add reload button to fail whale

### PR DESCRIPTION
**Pull request by rak-phillip**
This adds a button to the fail whale page to allow the user to "refresh" the page on error by attempting to navigate back one step in the router history. 

I attempted another approach to display the fail whale page without modifying the url by introducing a named catch-all route, similar to instructions detailed in [NUXT](https://nuxtjs.org/docs/2.x/configuration-glossary/configuration-router#extendroutes) and [Vue Router](https://next.router.vuejs.org/guide/essentials/dynamic-matching.html#catch-all-404-not-found-route), but I think the functionality that I was looking for is only available in Vue Router 4.x. I'm open to other potential solutions to resolve this issue.

#3421

### CODE REVIEW

**Comment by rak-phillip** [comment]
### New Fail Whale 
image

File: pages/fail-whale.vue [hunk 1]
```diff
@@ -73,4 +87,21 @@ export default {
...
+    text-align: center;
+    margin-top: 18px;
+    margin-bottom: 18px;
+    max-width: 450px;
+  }
+
+  hr:after {
+    background: var(--body-bg);
+    color: var(--body-text);
+    content: 'or';
```
Comment (nwmac) [inline pages/fail-whale.vue:101]: 'or' should be localized - you'll need to switch that from being rendered in the :after to do that

**Review by nwmac — CHANGES_REQUESTED** [review]
One thing needs localizing, other than that, looks good to go in.

**Review by nwmac — APPROVED** [review]
LGTM

**Pull request merged**